README.rst

```diff
@@ -118,6 +118,9 @@ dependencies on any operating system thanks to the
 `PyVISA-py <https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/>`__
 VISA backend. However, in order to use any VISA functionality that
 PyVISA-py does not implement, a third-party VISA backend such as
+`TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__
```

Comment:
This section is not valid syntax for rST. Please verify that the docs build locally.

Reply:
Run and Verified
![image](https://github.com/tektronix/tm_devices/assets/3426243/a701336f-9e3f-4ed1-9c7f-73017fab05ea)